lib/open_api_spex/cast/error.ex

```diff
@@ -71,10 +222,67 @@ defmodule OpenApiSpex.Cast.Error do
     end
   end
 
+  def message(%{reason: :all_of, value: value, meta: %{invalid_schema: invalid_schema}}) do
+    "Failed to cast #{inspect(value)} as #{invalid_schema}. Value must be castable using `allOf` schemas listed."
```

Comment:
I've been trying to avoid dumping the entire input value in the error message, because it's a potential DOS attack vector.

Reply:
Maybe a further discussion is in order.